### PR TITLE
Blog: Camel Quarkus 1.5.0

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@zbendhiba could you please review too?

**Review by aldettinger — APPROVED** [review]
Looks good to me. Well done.

**Comment by zbendhiba** [comment]
@ppalaga  is it normal to have 3000+ files changed ?

**Review by oscerd — CHANGES_REQUESTED** [review]
The blog post looks good, but the js files looks a bit suspicious

**Comment by davsclaus** [comment]
No something is wrong, a blog post is only the blog itself, and maybe some images.

**Comment by davsclaus** [comment]
Smells like you mistakenly added all temporary node files or something. I guess maybe we need to add more folders to gitignore or something.

**Comment by ppalaga** [comment]
Thanks for noticing, 26a7ea6 removes the unnecessary files.

**Review by zbendhiba — APPROVED** [review]

**Comment by zregvart** [comment]
We also need a [release note](https://github.com/apache/camel-website#sub-project-release) to make the release available on the download/release pages.

File: content/blog/2020/12/camel-quarkus-release-1.5.0/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+We are pleased to announce the release 1.5.0 of Camel Quarkus! What's inside?
+
+## Camel 3.7.0
+
+Camel Quarkus 1.5.0 is based on Camel 3.7.0 LTS bringing new features and improvements relevant to Camel Quarkus:
+
+### CSimple language
+
+[CSimple language](https://camel.apache.org/components/latest/languages/csimple-language.html) is a new variant of the well known [Simple language](https://camel.apache.org/components/latest/languages/simple-language.html). The "C" in its name stands for "compiled" and indeed, the expressions are compiled into Java byte code or native code at application build time. This brings significant performance boost at runtime. Camel Quarkus 1.5.0 brings experimental support for the CSimple language in both JVM and native modes.
```
Comment (davsclaus) [inline content/blog/2020/12/camel-quarkus-release-1.5.0/index.md:20]: You cannot use absolute url, start from /components/...

File: content/blog/2020/12/camel-quarkus-release-1.5.0/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Lightweight mode can be activated by adding the `camel-quarkus-main` dependency and adding
+
+```
+camel.main.lightweight=true
+```
+
+to `application.properties`.
+
+Check [Camel 3.7 what's new](https://camel.apache.org/blog/2020/12/Camel37-Whatsnew/) blog post by Claus Ibsen for more information about Camel 3.7.0.
```
Comment (davsclaus) [inline content/blog/2020/12/camel-quarkus-release-1.5.0/index.md:34]: Absolute urls to camel website itself will fail in build, see prev comment

**Review by davsclaus — CHANGES_REQUESTED** [review]

**Comment by ppalaga** [comment]
34a3ff0: 

* `s|https://camel.apache.org||`
* Added a release note

**Comment by ppalaga** [comment]
Is this good to merge now?

File: .gitignore [hunk 3]
```diff
@@ -20,3 +20,5 @@ resources
 !**/.yarn/sdks
 !**/.yarn/unplugged
 !**/.yarn/versions
+
+.pnp
```
Comment (zregvart) [inline .gitignore:24]: Not sure where this is created or why, but it sounds like we need it or it generates by error on your side. Either way I think it needs to be removed.

**Review by zregvart — CHANGES_REQUESTED** [review]

**Comment by zregvart** [comment]
[Download](https://deploy-preview-509--camel.netlify.app/download/) and [release](https://deploy-preview-509--camel.netlify.app/releases/q-1.5.0/) page look good.

**Comment by ppalaga** [comment]
Thanks for the review, everyone!

File: .gitignore (hunk 3 above)
Comment (ppalaga) [reply .gitignore:24]: Ups, sorry, I have not noticed this comment. You mean it should not be ignored?

Comment (ppalaga) [reply .gitignore:24]: I have force-pushed without this change to the master

Comment (zregvart) [reply .gitignore:24]: Yeah, I don't think it should be ignored, PNP stands for plug and play which is a Yarn feature that replaces node_modules, we also use another Yarn feature called zero install that vendors all the dependencies. The end result is, if you checkout the repository you don't need to fetch anything to build. For example, we don't need `yarn install` on the CI:

https://github.com/apache/camel-website/blob/33fedc838299b8b0b84aa24726ff813eb7ec5218/Jenkinsfile#L58

In that way it's odd that you get this file/directory created, as we also have a GitHub action that checks that everything is present in `.yarn` and in sync with `.pnp.js` and `yarn.lock`:

https://github.com/apache/camel-website/blob/33fedc838299b8b0b84aa24726ff813eb7ec5218/.github/workflows/pr.yaml#L28

Nevermind, I'll remove that from .gitignore.

Comment (ppalaga) [reply .gitignore:24]: > I'll remove that from .gitignore.

As I said, the change is not in the master anymore.

**Pull request merged**